api/tests/certification/complementary-certification/unit/application/attach-target-profile-route_test.js

```diff
@@ -6,47 +6,92 @@ import { expect, HttpTestServer, sinon } from '../../../../test-helper.js';
-        sinon.stub(securityPreHandlers, 'checkAdminMemberHasRoleSuperAdmin').returns(true);
-        const httpTestServer = new HttpTestServer();
-        await httpTestServer.register(moduleUnderTest);
-        const payload = {
-          data: {
-            attributes: {
-              'target-profile-id': 1,
-              'notify-organizations': false,
-              'complementary-certification-badges': [
-                {
-                  data: {
-                    attributes: {
-                      'badge-id': 1,
-                      level: 1,
-                      'image-url': 'imageUrl',
-                      label: 'label',
-                      'sticker-url': 'stickerUrl',
-                      'certificate-message': '',
-                      'temporary-certificate-message': '',
+      context('when there is a target profile to remove', function () {
+        it('should return a response with an HTTP status code 200', async function () {
+          // given
+          sinon.stub(attachTargetProfileController, 'attachTargetProfile').returns('ok');
+          sinon.stub(securityPreHandlers, 'checkAdminMemberHasRoleSuperAdmin').returns(true);
+          const httpTestServer = new HttpTestServer();
+          await httpTestServer.register(moduleUnderTest);
+          const payload = {
+            data: {
+              attributes: {
+                'target-profile-id': 1,
+                'notify-organizations': false,
+                'complementary-certification-badges': [
+                  {
+                    data: {
+                      attributes: {
+                        'badge-id': 1,
+                        level: 1,
+                        'image-url': 'imageUrl',
+                        label: 'label',
+                        'sticker-url': 'stickerUrl',
+                        'certificate-message': '',
+                        'temporary-certificate-message': '',
+                      },
+                      relationships: {},
+                      type: 'complementary-certification-badges',
                     },
-                    relationships: {},
-                    type: 'complementary-certification-badges',
                   },
-                },
-              ],
+                ],
+              },
             },
-          },
-        };
+          };
 
-        // when
-        const { statusCode } = await httpTestServer.request(
-          'PUT',
-          '/api/admin/complementary-certifications/1/badges',
-          payload,
-        );
+          // when
+          const { statusCode } = await httpTestServer.request(
+            'PUT',
+            '/api/admin/complementary-certifications/1/badges',
+            payload,
+          );
 
-        // then
-        expect(statusCode).to.equal(200);
+          // then
+          expect(statusCode).to.equal(200);
+        });
+      });
+      context('when there is no target profile to remove', function () {
```

Comment:
Petit saut de ligne pour aérer 😬 (fais moi savoir si ce genre de retour "relou" je peux le traiter moi même pour gagner du temps). Peut être le mieux serait d'avoir une règle eslint pour gérer ça automatiquement 🤷‍♂️

Reply:
sinon tu peux faire la suggestion, comme ca ont peut les prendre en compte via github (et un petit rebase)
```suggestion
      });

      context('when there is no target profile to remove', function () {
```